schemes/jellyfinScheme.ts

```diff
@@ -127,6 +127,8 @@ export default class JellyfinScheme {
     // Fetch the user, then set it in Nuxt Auth
     const user = (await this.$auth.ctx.app.$api.user.getCurrentUser()).data;
 
+    if (!user.Id) this.logout();
+
```

Comment:
On my production instance, I think that `param "userId" cannot be null or undefined` error is as a response to calling `/users/me` to the location of jf-vue instead of the location of jf-server. Therefore, it sends a response of the webpage, and a status code of 200, no errors here. By checking if the `Id` exists we should *hopefully* be able to fix most issues.

Reply:
However this doesn't fix the call right?

Reply:
No, but if the call fails, we will be logged out automatically, instead of being greeted with a blank homescreen and 
![image](https://user-images.githubusercontent.com/18101008/107116356-6ed1d700-686a-11eb-8025-194caf055bfd.png)